### PR TITLE
Upgrade dependencies for build and test

### PR DESCRIPTION
**Pull request by enyo**
Fixes jakemac53/built_html#8

### CODE REVIEW

**Comment by enyo** [comment]
Heads up: the test did not actually pass. For some reason the hash that was generated for me, was a different hash than the test expected.
I didn't want to simply change the hash to the one I got, because I don't know *why* it was different. I can't see my change causing this discrepancy though, so I assume that this test was already broken?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.2.0
```
Comment (jakemac53) [inline CHANGELOG.md]: The version should actually be `0.1.3` (since there is a leading zero the rest of the numbers are shifted to the right)

**Comment by jakemac53** [comment]
I pushed a commit to fix the test, build_runner just changed how it is computing digests a bit

File: CHANGELOG.md (hunk 1 above)
Comment (enyo) [reply CHANGELOG.md]: Changed.

**Review by jakemac53 — APPROVED** [review]

**Comment by jakemac53** [comment]
thanks @enyo I kept forgetting to do this :)

**Pull request merged**